### ISSUE TITLE
Shell completion creates empty log file

### ISSUE THREAD

**Issue by AndiDog** [xetys/hetzner-kube#300]
With v0.5.1:
```
$ cd /tmp && rm -rf /tmp/boom && mkdir /tmp/boom && cd /tmp/boom && hetzner-kube completion bash >/dev/null && ls -lh
total 0
-rw-r--r--  1 asommer  wheel     0B Dec  5 12:40 hetzner.log
```

I'm including the completion from bashrc, so whenever I split my terminal, the current working directory gets an empty hetzner.log file.

**Comment by xetys** [comment]
this is indirect because of #20

**Comment by xetys** [comment]
closed by 49203d500aed94a449061c532c28820b9dddabca
